wire-tests-swift/no-manifest/src/main/swift/AllTypes.swift

```diff
@@ -2009,216 +2009,216 @@ extension AllTypes {
-        public var default_uint32: Swift.UInt32?
-        @Wire.CustomDefaulted(defaultValue: Swift.Int32.min)
-        public var default_sint32: Swift.Int32?
-        @Wire.CustomDefaulted(defaultValue: Swift.UInt32.max)
-        public var default_fixed32: Swift.UInt32?
-        @Wire.CustomDefaulted(defaultValue: Swift.Int32.min)
-        public var default_sfixed32: Swift.Int32?
-        @Wire.CustomDefaulted(defaultValue: Swift.Int64.max)
-        public var default_int64: Swift.Int64?
-        @Wire.CustomDefaulted(defaultValue: Swift.UInt64.max)
-        public var default_uint64: Swift.UInt64?
-        @Wire.CustomDefaulted(defaultValue: Swift.Int64.min)
-        public var default_sint64: Swift.Int64?
-        @Wire.CustomDefaulted(defaultValue: Swift.UInt64.max)
-        public var default_fixed64: Swift.UInt64?
-        @Wire.CustomDefaulted(defaultValue: Swift.Int64.min)
-        public var default_sfixed64: Swift.Int64?
-        @Wire.CustomDefaulted(defaultValue: true)
-        public var default_bool: Swift.Bool?
-        @Wire.CustomDefaulted(defaultValue: 123.456e7)
-        public var default_float: Swift.Float?
-        @Wire.CustomDefaulted(defaultValue: 1.23456E80)
-        public var default_double: Swift.Double?
-        @Wire.CustomDefaulted(defaultValue: "\"çok\\u0007\\b\\u000c\\n\\r\\t\\u000b\\u0001\\u0001\\u0001\\u000f\\u000f~\\u0001\\u0001\\u0011\\u0001\\u0001\\u0011güzel\"")
-        public var default_string: Swift.String?
-        @Wire.CustomDefaulted(defaultValue: Foundation.Data(base64Encoded: "529rBwgMCg0JCwEBAQ8PfgEBEQEBEWf8emVs")!)
+        @CustomDefaulted(defaultValue: Int32.max)
+        public var default_int32: Int32?
+        @CustomDefaulted(defaultValue: UInt32.max)
+        public var default_uint32: UInt32?
+        @CustomDefaulted(defaultValue: Int32.min)
+        public var default_sint32: Int32?
+        @CustomDefaulted(defaultValue: UInt32.max)
+        public var default_fixed32: UInt32?
+        @CustomDefaulted(defaultValue: Int32.min)
+        public var default_sfixed32: Int32?
+        @CustomDefaulted(defaultValue: Int64.max)
+        public var default_int64: Int64?
+        @CustomDefaulted(defaultValue: UInt64.max)
+        public var default_uint64: UInt64?
+        @CustomDefaulted(defaultValue: Int64.min)
+        public var default_sint64: Int64?
+        @CustomDefaulted(defaultValue: UInt64.max)
+        public var default_fixed64: UInt64?
+        @CustomDefaulted(defaultValue: Int64.min)
+        public var default_sfixed64: Int64?
+        @CustomDefaulted(defaultValue: true)
+        public var default_bool: Bool?
+        @CustomDefaulted(defaultValue: 123.456e7)
+        public var default_float: Float?
+        @CustomDefaulted(defaultValue: 1.23456E80)
+        public var default_double: Double?
+        @CustomDefaulted(defaultValue: "çok\u{7}\u{8}\u{c}\n\r\t\u{b}\u{1}\u{1}\u{1}\u{f}\u{f}~\u{1}\u{1}\u{11}\u{1}\u{1}\u{11}güzel")
```

Comment:
note the removal of extra set of "" which were wrong. It also now correctly outputs unicode in the format the Swift compiler expects.